manifest.json

```diff
@@ -53,6 +53,19 @@
                     "fr": "Choisissez l’administrateur"
                 },
                 "example": "johndoe"
+            },
+            {
+                "name": "is_public",
+                "type": "boolean",
+                "ask": {
+                    "en": "Is it a public application?",
+                    "fr": "Est-ce une application publique ?"
+                },
+                "help": {
+                    "en": "It is strongly advised to keep the configuration UI private.",
+                    "fr": "Il est fortement conseillé de garder l'interface de configuration privée."
```

Comment:
What happens if you do make it public though ? Is there a use-case for it ? If we don't want people to make it public, do we really want to expose the option ?

Reply:
We can't pass any test except private one because the checks only look for something in public on non private checks

Reply:
Hmmmm I think I see what you mean ... Dunno what version of package_check you used but we fixed that ~recently in the new package_check version (still pending in a PR) : 

https://github.com/YunoHost/package_check/blob/cleanup-3/lib/tests.sh#L143-L148

So if the intention was really just to workaround the CI issue we should remove that setting xD

Reply:
Then should we wait for this new version before removing that workaround and merge ?

Reply:
> Hmmmm I think I see what you mean ... Dunno what version of package_check you used but we fixed that ~recently in the new package_check version (still pending in a PR) :
> 
> https://github.com/YunoHost/package_check/blob/cleanup-3/lib/tests.sh#L143-L148

Doesn't seem to work for restore/backup

Reply:
> Dunno what version of package_check you used
On my side, I was using master branch